### PR TITLE
Feat(ui): update displayName part 3

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes:

#11399

Added update displayName functionality for

- [x] Pipeline details page
- [x] mlModal details page
- [x] container details page
- [x] schema details page
- [x] database details page
- [x] service details page

https://github.com/open-metadata/OpenMetadata/assets/71748675/13d49a7b-b4e8-4b47-b7c1-b352dd30e21a

https://github.com/open-metadata/OpenMetadata/assets/71748675/b34d2cb4-52d4-4baa-8dd5-87140f58cd55

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [x] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx [hunk 1]
```diff
@@ -358,6 +359,31 @@ const ContainerPage = () => {
...
+
+      setContainerData((prev) => {
+        if (isUndefined(prev)) {
+          return;
+        }
+
+        return {
+          ...prev,
+          displayName,
+          version,
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx:380]: Any reason to update version too?

**Review by chirag-madlani — COMMENTED** [review]
@ShaileshParmar11 can you check why breadcrumbs not showing displayName?

File: openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx (hunk 1 above)
Comment (ShaileshParmar11) [reply openmetadata-ui/src/main/resources/ui/src/pages/ContainerPage/ContainerPage.tsx:380]: Yes we show updated version number on version button

**Comment by ShaileshParmar11** [comment]
> @ShaileshParmar11 can you check why breadcrumbs not showing displayName?

It will be part of next PR

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**